spirv_msl.cpp

```diff
@@ -965,6 +965,17 @@ void CompilerMSL::emit_entry_point_declarations()
 	}
 	// For some reason, without this, we end up emitting the arrays twice.
 	buffer_arrays.clear();
+
+	// Emit disabled fragment outputs.
+	for (uint32_t var_id : disabled_frag_outputs)
```

Comment:
Needs std::vector to ensure stable output (or std::map, but very few elements, so doesn't make much sense to use a hammer for this task)